.github/workflows/update.yaml

```
@@ -0,0 +1,32 @@
+name: Check for updates
+on:
+  schedule: # for scheduling to work this file must be in the default branch
+  - cron: "0 * * * *" # run every hour
```

```suggestion
    - cron: "0 * * * *" # run every hour
```

To keep the formatting consistent